Web_Development/FrontEnd/HTML_Commands/HML_Commands.md

```diff
@@ -1 +1,83 @@
-hi
\ No newline at end of file
+### The html tag: 
+The <html> tag points that the code is written in html and </html> tag states that it's the end of html code.
+
+### The head tag: 
+The head of an html code contains special information, for example: it's title. 
+
+```html 
+<html>
+   <head>
+      <title>Hello! Welcome to Girlscript Winter of Contributing</title>
+   </head>
+</html>
+```
+
+### Title: 
+A title tag specifies a title in your browser window.  
+```html
+<title>Hello! Welcome to Girlscript Winter of Contributing</title>
+```
+
+### The body tag: 
+The body tag specifies the beginning and end of the code that consiste of text, pictures, links, etc.
+```html
+<html>
+   <head>
+      <title>Hello! Welcome to Girlscript Winter of Contributing</title>
+   </head>
+   <body>
+```
+  
+### Headers: 
+There are up to six levels of headers that we can use in our html code, h1 through h6. Header 1 is the largest header and it decreases till header 6.
+
```

Comment:
The proper way to write the tags <h1> <h2>

Reply:
Changes done